server/ruby/Gemfile

```diff
@@ -1,6 +1,7 @@
 source 'https://rubygems.org/'
 
 gem 'dotenv'
+gem "rackup"
```

Comment:
`rackup` has been extracted as a gem from `rack` and Sinatra needs it.
📝 https://github.com/rack/rack/blob/main/CHANGELOG.md#user-content-removed